--- scons-4.11.0/SConstruct.omv~	2026-08-11 15:37:05.042559810 +0200
+++ scons-4.11.0/SConstruct	2026-08-11 15:37:05.043992080 +0200
@@ -206,7 +206,7 @@ wheel = env.Command(
         '$DISTDIR/SCons-${VERSION}.tar.gz',
     ],
     source=['pyproject.toml', 'setup.py', 'SCons/__init__.py'] + man_pages,
-    action='$PYTHON -m build --outdir $DISTDIR',
+    action='$PYTHON -m build --no-isolation --outdir $DISTDIR',
 )
 env.Alias("wheel", wheel[0])
 env.Alias("tar-gz", wheel[1])
